tests/kyverno-native/kyverno-test.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: cli.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: prepend-cluster-app-config-map-test
+policies:
+  - ../../policies/ux/prepend-cluster-app-config-map.yaml
+resources:
+  - prepend-cluster-app-config-map/app-pre.yaml
+targetResources:
+  - prepend-cluster-app-config-map/app-pre.yaml
+results:
+  # BUG: using any part of `mutate.target` in the ClusterPolicy makes this fail
```

Comment:
Can we remove this "leftover"? It's not actionable.

Also, the tests should cover all cases:
- Entry exists and is identical (idempotent or no-op)
- Entry exists and differs
- Entry does not exist